src/pages/AdminSettings/AdminMembers/MemberList/MemberTable.tsx

```diff
@@ -0,0 +1,237 @@
+import {
+  UseMutateFunction,
+  useMutation,
+  useQueryClient,
+} from '@tanstack/react-query'
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import cs from 'classnames'
+import { useEffect, useMemo } from 'react'
+import { useInView } from 'react-intersection-observer'
+
+import { useLocationParams } from 'services/navigation'
+import {
+  SelfHostedSettings,
+  UserListOwner,
+  useSelfHostedSettings,
+  useSelfHostedUserList,
+} from 'services/selfHosted'
+import Api from 'shared/api'
+import Spinner from 'ui/Spinner'
+import Toggle from 'ui/Toggle'
+
+import 'ui/Table/Table.css'
+
+type MembersColumns = {
+  username: React.ReactNode
+  type: string
+  email: React.ReactNode
+  activationStatus: React.ReactNode
+}
+
+const columnHelper = createColumnHelper<MembersColumns>()
+
+const columns = [
+  columnHelper.accessor('username', {
+    header: 'User Name',
```

Comment:
While we're here, should we update this to Username? there's one more copy reference in AdminTable.jsx too. 

Thought here is github calls it a Username, we should keep the same semantics

Reply:
I believe @spalmurray-codecov was just updating the `AdminTable`, if he wants to make the update there 👀